conanfile.py

```diff
@@ -49,6 +49,19 @@ def source(self):
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()''')
 
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              """if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
This seems a bit of a hack TBH, it will break if CMakeLists.txt changes in these lines, even slightly.
It's probably better to modify CMakeLists.txt, wrapping this code in an if that checks if the CMake variable FRUIT_IS_BEING_BUILT_BY_CONAN is not defined.

Reply:
You are right.
I will force push.

Reply:
The my last commit works on my machine (Windows 10 with Visual Studio 2019).
However, I missed a problem of building shared library (#105) .